src/js/modules/book.js

```diff
@@ -6,8 +6,9 @@ const mainBook = {
     })
     document.getElementById('pageSlider').addEventListener('change', () => {

```

Comment:
Using `getElementById`? Other times we use querySelectorAll and querySelector. We should use one of them...